### PR TITLE
completed #6 GET /api/reviews/:review_id/comments

### PR DESCRIPTION
**Pull request by tyrese-s**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -118,7 +118,49 @@ describe('app', () => {
             .then((response) => {
                 const responseMessage = response.body.msg
                 expect(responseMessage).toBe('invalid input id')
+            });
+        });
+    })
```
Comment (HannahBurks) [inline __tests__/app.test.js:123]: We also want to add a test here (200) - that if the ID exists but there is no comments that it returns an empty array.

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Hey Tyrese, very nice work! Just one extra test to add in this case, once added please send another PR 😄

File: __tests__/app.test.js [hunk 2]
```diff
@@ -118,7 +118,66 @@ describe('app', () => {
...
+        test('400: responds with 400 and msg when incorrect id input', () => {
+            return request(app)
+            .get('/api/reviews/anything/comments')
+            .expect(400)
+            .then((response) => {
+                const responseMessage = response.body.msg
+                expect(responseMessage).toBe('invalid input id')
+            });
+        });
+    })
```
Comment (jimstev2447) [inline __tests__/app.test.js:166]: you've got a test for invalid id but are missing one for valid but non existent e.g /api/reviews/1000/comments
I would recommend add ing it in here

File: __tests__/app.test.js [hunk 3]
```diff
@@ -118,7 +118,66 @@ describe('app', () => {
...
+    // describe('POST/api/reviews/:review_id/comments', () => {
```
Comment (jimstev2447) [inline __tests__/app.test.js:167]: try to avoid leaving in commented out code it can make merge conflicts more likely and you dont know if its needed yet :)

**Comment by cjplatten** [comment]
When adding files to a commit don't just add every file by default, instead only add the files that are relevent to the commit. There are changes included in this commit that are for a different ticket entirely, this makes it harder to review and could cause you problems if you ever needed to revert to a previous commit. 

Otherwise, this is ok to merge.
